src/stabilities/styles.tsx

```diff
@@ -0,0 +1,186 @@
+import { makeStyles } from '@material-ui/styles';
+import { getMinMediaQuery } from './utils';
+
+export const useGetStyles = () => {
+  const useStyles = makeStyles({
+    root: {
+      maxWidth: '100%',
+      height: 'auto',
+      display: 'flex',
+      borderRadius: '2px',
+      overflow: 'scroll',
+      '& h1': {
+        fontSize: '1.375rem',
+        fontWeight: 500,
+        margin: '0',
+        marginBottom: '1rem',
+        [getMinMediaQuery(1600)]: {
+          marginBottom: '1.5rem',
+        },
+      },
+      '& .itemTitle': {
+        fontSize: '0.75rem',
+        fontWeight: 300,
+        margin: '0',
+        color: '#777777',
+        '& .marginTopPercentage': {
+          marginTop: '8%',
+        },
+        [getMinMediaQuery(1600)]: {
+          fontSize: '0.875rem',
+        },
+      },
+      '& .amount': {
+        fontSize: '1rem',
+        fontWeight: 400,
+        margin: '0rem 0 0 0rem',
+        color: '#000000',
+        '& .marginBottom': {
+          marginBottom: '2rem',
+          maxMarginBottom: '1.5rem',
+        },
+        [getMinMediaQuery(1600)]: {
+          fontSize: '1.25rem',
+        },
+      },
+      '& .amountInChart': {
+        fontSize: '1rem',
+        fontWeight: 400,
+        margin: '0',
+        height: '1.65rem',
+        color: '#000000',
+        [getMinMediaQuery(1600)]: {
+          fontSize: '1.25rem',
+        },
+      },
+      '& .inChart': {
+        height: '1rem',
+        marginLeft: '0rem',
+        marginTop: '0rem',
+        fontSize: '0.5rem',
+        [getMinMediaQuery(1600)]: {
+          fontSize: '0.875rem',
+        },
+      },
+      '& .line': {
+        height: '0.0625rem',
+        width: 'auto',
+        background: '#E8E8E8',
+        margin: '1.5rem 0 1.5rem 0',
+      },
+    },
+    container: {
+      height: 'auto',
+      width: '100%',
+      margin: '1.5rem 1rem 1.5rem 1rem',
+      [getMinMediaQuery(1600)]: {
+        margin: '2rem 1.5rem 2rem 1.5rem',
+      },
+    },
+    chartCenter: {
+      width: 'height',
+      height: '99%',
+      margin: '3.3rem 0 0 0',
+      position: 'absolute',
+      textAlign: 'center',
+    },
+    coinInChart: {
+      fontWeight: 400,
+      margin: '0',
+      height: '1rem',
+      marginLeft: '0rem',
+      marginTop: '0rem',
+      fontSize: '0.75rem',
+      [getMinMediaQuery(1600)]: {
+        fontSize: '0.875rem',
+      },
+    },
+    atom: {
```

Comment:
We should not be naming this atom since this component will not just be servicing cosmos

Reply:
changed to "coin"